src/SME.SGP.WebClient/public/configuracoes/variaveis.json

```diff
@@ -1,4 +1,4 @@
 {
-  "API_URL": "https://hom2-novosgp.sme.prefeitura.sp.gov.br/api",
+  "API_URL": "https://localhost:5001/api",
```

Comment:
Faz diferença se for o server de execução local e não o de homologação?

Reply:
não

Reply:
OK.